test/jdk/tools/jpackage/helpers/jdk/jpackage/test/Functional.java

```diff
@@ -158,6 +158,10 @@ public static void rethrowUnchecked(Throwable throwable) throws ExceptionBox {
             new ExceptionBox(throwable.getCause());
         }
 
+        if (throwable.getClass().getName().equals("jtreg.SkippedException")) {
```

Comment:
Would it make sense to have check: `if (throwable instanceof Runnable)`?

Reply:
Not sure. I do not think it will work. SkippedException extends RuntimeException, so not sure why we need to check it with Runnable.

Reply:
Not sure. I do not think it will work. SkippedException extends RuntimeException, so not sure why we need to check it with Runnable.

Reply:
My point is that if the exception is Runnable, there is no need to wrap it in ExceptionBox instance.